phpBB/composer.json

```diff
@@ -33,7 +33,7 @@
 		"marc1706/fast-image-size": "1.1.*",
 		"paragonie/random_compat": "^1.2",
 		"patchwork/utf8": "1.1.*",
-		"s9e/text-formatter": "^0.4.2",
+		"s9e/text-formatter": "^0.5.0",
```

Comment:
Unfortunately I have no idea what's the difference between those operators. Since my intention was to update the version number without changing the semantics of the operator it is associated with, I will withdraw this pull request and let you sort it out at your convenience. This seems much more efficient this way.